includes/core.php

```diff
@@ -780,6 +780,7 @@ function get_available_design_styles() {
 					'tertiary'          => '#f7fbff',
 					'background'        => '#ffffff',
 					'header_background' => '#3f46ae',
+					'header_text_color' => '#fafafa',
```

Comment:
This needs to be `header_text` for it to function correctly. Switching this to `header_text_color` causes the issue to persist.